Assets/HoloToolkit/Input/Scripts/Focus/FocusManager.cs

```diff
@@ -119,6 +120,20 @@ private class PointerData
         {
             public readonly IPointingSource PointingSource;
 
+            private PointerInputEventData pointerData;
+            public PointerInputEventData UnityUIPointerData
+            {
+                get
+                {
+                    if (pointerData == null)
+                    {
+                        pointerData = new PointerInputEventData(EventSystem.current);
```

Comment:
This also looks familiar, except the previous implementation called `Clear(uiRaycastPointerData);` (in this case `pointerData`) before returning if it wasn't null. Should we do that here?

Reply:
No, we should only be clearing it before we update our UnityUIRaycasts.